tools/sil-opt/SILOpt.cpp

```diff
@@ -104,6 +104,10 @@ static llvm::cl::opt<bool>
 EnableExperimentalConcurrency("enable-experimental-concurrency",
                    llvm::cl::desc("Enable experimental concurrency model."));
 
+static llvm::cl::opt<bool> EnableExperimentalLexicalLifetimes(
+    "enable-experimental-lexical-lifetimes",
```

Comment:
Any option that's not in the frontend is naturally experimental. You can save some command line typing here.

Reply:
As discussed, we're going to keep this name just to match the frontend flag's name.